cpp/src/traversal/opg/bfs_comms.cuh

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <raft/handle.hpp>
+#include <graph.hpp>
+
+
+namespace detail {
+
+namespace opg {
```

Comment:
Are we plan to use the opg namesapce?

Reply:
And the functions in the opg namespace are not really opg specific.

divup below resembles

https://github.com/rapidsai/cudf/blob/7f91a17a967b0e0b502d56d37818aa0461d38c67/cpp/include/cudf/detail/utilities/integer_utils.hpp#L77

from cuDF.

To maximize code duplication/code reuse, I don't think bfs_comms.cuh is the right file to define these functions. Ideally, this should go to RAFT.